matchya-web/src/data/index.ts

```diff
@@ -0,0 +1,25 @@
+import { Score } from '../types';
```

Comment:
Let's name the folder 'mock'

Reply:
Should I rename it from types to mock? or should I rename the data folder to mock?